packages/manager/modules/pci/src/projects/project/private-networks/private-networks.html

```diff
@@ -67,49 +66,113 @@
             </button>
         </oui-datagrid-topbar>
         <oui-datagrid-column
-            data-title=":: 'pci_projects_project_network_private_name' | translate"
-            data-property="name"
+            data-title=":: 'pci_projects_project_network_private_vlan_id' | translate"
+            data-property="vlanId"
             data-searchable
             data-type="string"
             data-filterable
         ></oui-datagrid-column>
         <oui-datagrid-column
-            data-title=":: 'pci_projects_project_network_private_id' | translate"
-            data-property="id"
-            data-type="string"
-            data-filterable
-        ></oui-datagrid-column>
-        <oui-datagrid-column
-            data-title=":: 'pci_projects_project_network_private_region' | translate"
-            data-property="formattedRegions"
-            data-type="string"
+            data-title=":: 'pci_projects_project_network_private_name' | translate"
+            data-property="name"
             data-searchable
-            data-filterable
-        >
-            <span data-ng-bind="$row.regions.length"></span>
-            <button
-                class="oui-popover-button"
-                data-oui-popover="{{ $row.formattedRegions }}"
-                type="button"
-            ></button>
-        </oui-datagrid-column>
-        <oui-datagrid-column
-            data-title=":: 'pci_projects_project_network_private_vlan_id' | translate"
-            data-property="vlanId"
             data-type="string"
-            data-searchable
             data-filterable
         ></oui-datagrid-column>
-        <oui-datagrid-column
-            data-title=":: 'pci_projects_project_network_private_address' | translate"
-            property="address"
-        ></oui-datagrid-column>
-        <oui-action-menu data-compact data-placement="end">
-            <oui-action-menu-item on-click="$ctrl.deleteNetwork($row.id)">
-                <span
-                    data-translate="pci_projects_project_network_private_delete"
-                ></span>
-            </oui-action-menu-item>
-        </oui-action-menu>
+        <oui-datagrid-row-detail>
+            <div
+                class="oui-table-responsive grid-details-table"
+                data-ng-if="$row.subnet.length > 0"
+            >
```

Comment:
**suggestion:** Can you check if we need to show any message if subnets are not present?